fdbserver/fdbserver.actor.cpp

```diff
@@ -1488,6 +1488,11 @@ int main(int argc, char* argv[]) {
 		// evictionPolicyStringToEnum will throw an exception if the string is not recognized as a valid
 		EvictablePageCache::evictionPolicyStringToEnum(flowKnobs->CACHE_EVICTION_POLICY);
 
+		if (memLimit < FLOW_KNOBS->PAGE_CACHE_4K + (2LL<<30)) {
```

Comment:
I think there may be some people who intentionally want to run low memory clusters, which may conceivably function with less than 2GiB over page cache (possibly contingent on other knobs). We may want to reduce the required extra amount to allow for that.

Reply:
I can also just test that memLimit is at least strictly larger? I think that might be good enough. The problem seems to be that some people might think that `memory` and `cache_memory` are two different pools and I mostly want to defend against that